src/main/java/ch/jalu/configme/properties/ArrayProperty.java

```diff
@@ -0,0 +1,127 @@
+package ch.jalu.configme.properties;
+
+import ch.jalu.configme.properties.types.BeanPropertyType;
+import ch.jalu.configme.properties.types.PropertyType;
+import ch.jalu.configme.resource.PropertyReader;
+
+import javax.annotation.Nullable;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class ArrayProperty<T> extends BaseProperty<T[]> {
```

Comment:
Do we actually need an array type? When is it interesting to use this vs. a `List<T>` property?

Reply:
I am use array in next case:
`player.sendMessage(settingsManager.getProperty(ARRAY_PROPERTY))`
Player have a methods `sendMessage(String message)` and `sendMessage(String[] message)`

Reply:
OK for now :) I wanted to make sure that the array type is introduced for a reason; this is a good reason.